fhirAssembly/src/test/java/io/vantiq/extsrc/fhirAssembly/TestFhirUseAccessToken.java

```diff
@@ -0,0 +1,304 @@
+                    (Map<String, String>) mapper.readValue(new Gson().toJson(((JsonObject) resp.getBody())),
+                                                                                      Map.class);
+            log.debug("Found headers: {}", foundHeaders);
+            assertNotNull("Ho headers found", foundHeaders);
+            checkAuthHeader(tType, tValue, foundHeaders);
+            
+            Map<String, String> params = new HashMap<>();
+            
+            params.put("id", "doesn'tMatter");
+            params.put("type", "Patient");
+            resp = v.execute("com.vantiq.fhir.fhirService.read", params);
+            assertTrue("Could not read resource: " + resp.getErrors(), resp.isSuccess());
+            //noinspection unchecked
+            foundHeaders =
+                    (Map<String, String>) mapper.readValue(new Gson().toJson(((JsonObject) resp.getBody())),
+                                                           Map.class).get("body");
+            log.debug("Found headers: {}", foundHeaders);
+            
+            assertNotNull("Ho headers found", foundHeaders);
+            checkAuthHeader(tType, tValue, foundHeaders);
+            
+            File ourHeroFile = new File("build/resources/test/ourHero.json");
+            Map<String, ?> ourHeroMap = null;
+            try {
+                //noinspection unchecked
+                ourHeroMap = mapper.readValue(ourHeroFile, Map.class);
+            } catch (Exception e) {
+                fail("Couldn't read standard file" + e);
+            }
+            
+            resp = v.execute("com.vantiq.fhir.fhirService.create",
+                             Map.of("type", "Patient", "resource", ourHeroMap));
+            assertTrue("Could not create our hero: " + resp.getErrors(), resp.isSuccess());
+            //noinspection unchecked
+            foundHeaders =
+                    (Map<String, String>) mapper.readValue(new Gson().toJson(((JsonObject) resp.getBody())),
+                                                           Map.class).get("body");
+            log.debug("Found headers: {}", foundHeaders);
+            
+            assertNotNull("Ho headers found", foundHeaders);
+            checkAuthHeader(tType, tValue, foundHeaders);
+            
+            Map<String, ?> modifiers = Map.of("headers",
+                                              Map.of("If-None-Exist", "name=" + "Man"));
+            
+            // Now, create our here again, but conditional on it's not being already there.
+            
+            resp = v.execute("com.vantiq.fhir.fhirService.create",
+                             Map.of("type", "Patient",
+                                    "id", "batman",
+                                    "resource", ourHeroMap,
+                                    "modifiers", modifiers));
+            assertTrue("Could not create our hero: " + resp.getErrors(), resp.isSuccess());
+            //noinspection unchecked
```

Comment:
The call above returns an Object that contains the status code, body, and any response headers.  So we need to get the body part of that.

The remaining FHIR calls in this method operate the same way.